prisma/schema.prisma

```diff
@@ -38,8 +38,16 @@ model Tournament {
 }
 
 model Tag {
-  name        String        @unique
-  venues      Venues[]
-  meets       Meets[]
-  tournaments Tournaments[]
+  name        String       @unique
+  venues      Venue[]
+  meets       Meet[]
+  tournaments Tournament[]
+}
+
+model Venue {
```

Comment:
Error: Error validating model "Venue": Each model must have at least one unique criteria that has only required fields.

Please fix.